packages/desktop-client/src/components/table.tsx

```diff
@@ -197,11 +197,6 @@ export function Cell({
         privacyFilter={mergeConditionalPrivacyFilterProps(
           {
             activationFilters: [!focused, !exposed],
-            style: {
-              position: 'absolute',
-              width: '100%',
-              height: '100%',
-            },
```

Comment:
This appears to be safe to remove. But I'm also not clear why it's needed in current `master`. So I may be missing some edge case, and this might need to be reverted.

Reply:
Aha the e2e tests failed, maybe related. Apologies for pushing up failing code, still getting to know this project's dev workflow. Looking into it